tests/doneahuerta_3/screen-output.tmp

```diff
@@ -0,0 +1,29 @@
+-----------------------------------------------------------------------------
+-- This is ASPECT, the Advanced Solver for Problems in Earth's ConvecTion.
+--     . version 2.1.0-pre (DoneaHuertaBench, e84c31e)
+--     . using deal.II 8.5.1
+--     . using Trilinos 12.10.1
+--     . using p4est 2.0.0
+--     . running in DEBUG mode
+--     . running with 1 MPI process
+-- How to cite ASPECT: https://aspect.geodynamics.org/cite.html
+-----------------------------------------------------------------------------
+
+Loading shared library <./libdoneahuerta.so>
+ERROR: Uncaught exception in MPI_InitFinalize on proc 0. Skipping MPI_Finalize() to avoid a deadlock.
+
+
+----------------------------------------------------
+Exception 'ExcMessage (std::string("Could not successfully load shared library <") + shared_libs_list[i] + ">. The operating system reports " + "that the error is this: <" + dlerror() + ">.")' on rank 0 on processing: 
```

Comment:
This isn't quite right -- you'll need to add `doneahuerta.cc` that simply does a `#include "..."` to the corresponding file in the benchmarks folder.

Reply:
done